### DISCUSSION TITLE
Wasabi S3

### DISCUSSION THREAD

**Discussion by tobyworks** [pocketbase/pocketbase#1500] [Q&A] [answered]
Would like to suggest to be able to manually enter the S3 compatible URL in the fields of files Storage. Because whatever i do in which order i cant seem to connect pocketbase with my Wasabi S3 Bucket

I have tried these settings. 

doubled check the bucket name and access key are correct.

<img width="573" alt="image" src="https://user-images.githubusercontent.com/2817395/210862259-e4efa505-1d96-43ec-ba69-2690a98c1f19.png">

**Comment by ganigeorgiev** [comment, marked as answer]
I've just created a trial Wasabi account and it works fine for me when using the full bucket endpoint from the dashboard, aka. if your bucket is in the `eu-west-1` region, then the `Endpoint` address should be `s3.eu-west-1.wasabisys.com` (my test was with a bucket in `eu-central-1` and root account key).

**Reply by tobyworks** [reply]
Thank you so much
